Master-OB-OpenAPI.json

```
@@ -6989,6 +6998,254 @@
+              "Unit": "",
+              "Value": ""
+            },
+            "x-ob-item-type-group": ""
+          }
+        ]
+      },
+      "RoofPlaneID": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementString"
+          },
+          {
+            "type": "object",
+            "description": "The uuid of the roofplane",
+            "x-ob-item-type": "solar-types:uuidItemType",
+            "x-ob-usage-tips": "",
+            "x-ob-sample-value": {
+              "Decimals": "",
+              "EndTime": "",
+              "Precision": "",
+              "StartTime": "",
+              "Unit": "",
+              "Value": ""
+            },
+            "x-ob-item-type-group": ""
+          }
+        ]
+      },
+      "PVSystemArray": {
+        "type": "object",
+        "description": "An array of modules within a PVSystem",
+        "properties": {
+          "PVSystemStrings": {
+            "$ref": "#/components/schemas/PVSystemStrings"
+          },
+          "Orientation": {
+            "$ref": "#/components/schemas/Orientation"
+          },
+          "Area": {
+            "$ref": "#/components/schemas/Area"
+          },
+          "RoofPlaneID": {
+            "$ref": "#/components/schemas/RoofPlaneID"
+          },
+          "FileFolderURL": {
+            "$ref": "#/components/schemas/FileFolderURL"
+          },
+          "Description": {
+            "$ref": "#/components/schemas/Description"
+          },
+          "CapacityDC": {
+            "$ref": "#/components/schemas/CapacityDC"
+          },
+          "ProductLineItems": {
+            "$ref": "#/components/schemas/ProductLineItems"
+          },
+          "Devices": {
+            "$ref": "#/components/schemas/Devices"
+          }
+        }
+      },
+      "PVSystemStrings": {
+        "type": "array",
+        "items": {
+          "$ref": "#/components/schemas/PVSystemString"
+        }
+      },
+      "PVSystemString": {
+        "type": "object",
+        "description": "A string of Modules within a PVSystem",
+        "properties": {
+          "Devices": {
+            "$ref": "#/components/schemas/Devices"
+          },
+          "Orientation": {
+            "$ref": "#/components/schemas/Orientation"
+          },
+          "ProductLineItems": {
+            "$ref": "#/components/schemas/ProductLineItems"
+          },
+          "CapacityDC": {
+            "$ref": "#/components/schemas/CapacityDC"
+          },
+          "FileFolderURL": {
+            "$ref": "#/components/schemas/FileFolderURL"
+          },
+          "Description": {
+            "$ref": "#/components/schemas/Description"
+          },
+          "PVSystemStringID": {
+            "$ref": "#/components/schemas/PVSystemStringID"
+          }
+        }
+      },
+      "Orientation": {
+        "description": "Azimuth is also called Yaw. Tilt is also called Pitch, and Roll is called Roll.",
```

```suggestion
        "description": "Orientation is defined by Azimuth, Tilt and Roll. Azimuth and Tilt are defined in the context of a right-hand coordinate system where the the x-axis is East, y-axis is North, and the z-axis is vertical."
```

I don't think we should provide the synonyms because they suggest that Yaw and Pitch are also OB elements.